pkg/globalnet/controllers/global_ingressip_controller.go

```diff
@@ -165,29 +162,47 @@ func (c *globalIngressIPController) onCreate(ingressIP *submarinerv1.GlobalIngre
 	}
 
 	if ingressIP.Spec.Target == submarinerv1.ClusterIPService {
-		chainName := ingressIP.GetAnnotations()[kubeProxyIPTableChainAnnotation]
-		if chainName == "" {
+		serviceRef := ingressIP.Spec.ServiceRef
+
+		service, exists, err := getService(serviceRef.Name, ingressIP.Namespace, c.services, c.scheme)
+		if err != nil || !exists {
 			_ = c.pool.Release(ips...)
+			key := fmt.Sprintf("%s/%s", ingressIP.Namespace, serviceRef.Name)
+			klog.Warningf("Exported Service %q does not exist yet - re-queueing", key)
```

Comment:
We should log the error.

```suggestion
                         if err != nil {
                             klog.Errorf("Error retrieving exported Service %q - re-queueing", key)
                         } else {
		            klog.Warningf("Exported Service %q does not exist yet - re-queueing", key)
		        }
```